README.md

```diff
@@ -1,18 +1,21 @@
-## `ion-cli`
+# `ion-cli`
+
 This repository is home to the `ion` command line tool, which provides subcommands
 for working with [the Ion data format](https://amzn.github.io/ion-docs/docs/spec.html).
 
 ## Table of contents
+
 * [Examples](#examples)
-   * [Converting between formats with `dump`](#converting-between-formats-with-dump)
-   * [Converting between Ion and other formats with `to` and `from`](#converting-between-ion-and-other-formats-with-to-and-from)
-   * [Analyzing binary Ion file encodings with `inspect`](#analyzing-binary-ion-file-encodings-with-inspect)
+    * [Viewing the contents of an Ion file](#viewing-the-contents-of-an-ion-file)
+    * [Converting between Ion formats](#converting-between-ion-formats)
+    * [Converting between Ion and other formats with `to` and `from`](#converting-between-ion-and-other-formats-with-to-and-from)
+    * [Analyzing binary Ion file encodings with `inspect`](#analyzing-binary-ion-file-encodings-with-inspect)
 * [Installation](#installation)
-   * [via `brew`](#via-brew)
-   * [via `cargo`](#via-cargo)
+    * [via `brew`](#via-brew)
+    * [via `cargo`](#via-cargo)
```

Comment:
🗺️ My IDE modified the spacing for the bullet lists. Hard to be mad about it though; GitHub markdown's default list indentation is _three_ spaces. 🤨